modules/installation-creating-gcp-iam-shared-vpc.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+
+[id="installation-creating-gcp-iam-shared-vpc_{context}"]
+= Creating IAM roles in GCP
```

Comment:
As HOST_PROJECT_CONTROL_SUBNET and HOST_PROJECT_COMPUTE_SUBNET are defined in Creating VPC section, please remove them from this section.

Reply:
I only see where these variables are used in this file now, so if you're looking through the preview, it might not have refreshed right. I'm uploading a fresh preview now.